gwen/models/scrub.py

```diff
@@ -84,9 +84,16 @@ def hash_string(self, value):
         # taking least sig digits isn't guaranteed unique - use more as needed
         digits = 2
         full_hash = str(hash(value))
+
+        def first_letter(letter):
```

Comment:
Would it make more sense for this function to operate on a string, rather than a single character?

Reply:
not really, that's the point of the outer scope function.